classes/Category.php

```diff
@@ -2441,4 +2441,15 @@ public function existsInShop($idShop)
 		WHERE `id_category` = ' . (int) $this->id . '
 		AND `id_shop` = ' . (int) $idShop, false);
     }
+
+    /**
+     * Indicates whether a category is ROOT for the shop.
+     * The root category is the one with no parent. It's a virtual category.
+     *
+     * @return bool
+     */
+    public function isRootCategory(): bool
+    {
+        return 0 === (int) $this->id_parent;
```

Comment:
`isRootCategoryForAShop` already exists.

Reply:
isRootCategoryForAShop returns the home category, not the root

Reply:
You probably have to change the docstring of `isRootCategoryForAShop` method, it's confusing, description are similar :sweat_smile: 

There are also some call in `BulkDeleteCategoriesHandler` and `DeleteCategoryHandler` that are related to Root category, I'm not sure they thought about it. (Not in this pull request of course)

Reply:
Yes there are a lot of cinfusions between Home category of a Shop and the Root category which is a "virtual category"

Reply:
Could you change the description of the `isRootCategoryForAShop` by setting `Home category` (without changing the method signature of course)?